tests/slo/src/runner.py

```diff
@@ -121,7 +121,6 @@ def run_from_args(args):
     driver_config = ydb.DriverConfig(
         args.endpoint,
         database=args.db,
-        credentials=ydb.credentials_from_env_variables(),
```

Comment:
why have you deleted this? previous SLO was also with anonymous credentials

Reply:
Because it failed

Reply:
Everything was fine with previous SLO. 
Probably there's something wrong with new YDB cluster? It is not good to hide the issue